### PR TITLE
MLH- 599

### PR DESCRIPTION
**Pull request by ivansh83**
Created a function nameFieldInput in the .js file with the same name in the directory 'helpers'; created TC-084 and updated TC-083 using the function.

### CODE REVIEW

**Review by Ekatim — APPROVED** [review]

File: test/regression/name.js [hunk 1]
```diff
@@ -18,16 +17,14 @@ describe('Name field verification', function () {
         nameField.addValue(testData.minAge);
```
Comment (JuliaLobanova) [inline test/regression/name.js:17]: nameField -from where it's coming?

File: test/regression/name.js [hunk 2]
```diff
@@ -18,16 +17,14 @@ describe('Name field verification', function () {
...
 
     it('TC-083 Verify that the Name input field accepts letters', function () {
-        const input = $(sel.name);
-        input.doubleClick();
-        browser.keys("Delete");
-        input.setValue(testData.name);
-        const value = input.getValue();
-        expect(value).toEqual(testData.name);
+        nameFieldInput(testData.name);
+    });
```
Comment (JuliaLobanova) [inline test/regression/name.js:24]: no expect, what test is about?

File: test/regression/name.js [hunk 3]
```diff
@@ -18,16 +17,14 @@ describe('Name field verification', function () {
...
 
+    it('TC-083 Verify that the Name input field accepts numbers', function () {
+        nameFieldInput(testData.age);
     });
```
Comment (JuliaLobanova) [inline test/regression/name.js:28]: no expect

File: test/regression/name.js [hunk 4]
```diff
@@ -18,16 +17,14 @@ describe('Name field verification', function () {
...
-        input.doubleClick();
-        browser.keys("Delete");
-        input.setValue(testData.name);
-        const value = input.getValue();
-        expect(value).toEqual(testData.name);
+        nameFieldInput(testData.name);
+    });
 
+    it('TC-083 Verify that the Name input field accepts numbers', function () {
+        nameFieldInput(testData.age);
```
Comment (JuliaLobanova) [inline test/regression/name.js:27]: no expect

File: helpers/nameFiledInput.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+const sel = require('../data/selectors.json');
+function nameFieldInput(testData) {
+    const input = $(sel.name);
+    input.doubleClick();
+    browser.keys("Delete");
+    input.setValue(testData);
+    const value = input.getValue();
+    expect(value).toEqual(testData);
```
Comment (JuliaLobanova) [inline helpers/nameFiledInput.js:8]: in helper no expect, this is a function, not test

**Review by JuliaLobanova — CHANGES_REQUESTED** [review]
see my comments

File: helpers/nameFiledInput.js (hunk 5 above)
Comment (ivansh83) [reply helpers/nameFiledInput.js]: Please check the updates.

File: test/regression/name.js (hunk 4 above)
Comment (ivansh83) [reply test/regression/name.js]: Please check the updates.

File: test/regression/name.js (hunk 3 above)
Comment (ivansh83) [reply test/regression/name.js:38]: Please see the updates.

**Review by JuliaLobanova — APPROVED** [review]

**Pull request merged**